sagify/api/initialize.py

```diff
@@ -25,18 +26,18 @@ def _template_creation(app_name, aws_profile, aws_region, python_version, output
     Path(output_dir).mkdir(exist_ok=True)
     Path(os.path.join(output_dir, '__init__.py')).touch()
 
-    cookiecutter(
-        template=os.path.join(_FILE_DIR_PATH, '../template/'),
-        output_dir=output_dir,
-        no_input=True,
-        extra_context={
-            "project_slug": app_name,
-            "module_slug": sagify_module_name,
-            "aws_profile": aws_profile,
-            "aws_region": aws_region,
-            "python_version": python_version
-        }
-    )
+    # Set 'sagify module' directory up
```

Comment:
Instead of `cookiecutter` going through the template, we simply cp* the sagify directory as-is now.